### PR TITLE
[TIDOC-2611] Since 5.5.0.GA iOS platform return iOS not iPhone OS

### PR DESCRIPTION
**Pull request by HazemKhaled**
**JIRA:** https://jira.appcelerator.org/browse/AC-4460

### CODE REVIEW

File: apidoc/Titanium/Platform/Platform.yml [hunk 1]
```diff
@@ -208,6 +208,8 @@ properties:
     summary: |
         Name of the platform. Returns `android` for the Android platform, `iPhone OS` for the iOS
         platform (iPhone, iPad, or iPod Touch), `windows` for the Windows platform, and `mobileweb` for the Mobile Web platform.
+        
+        Since 5.5.0.GA return 'iOS' for for the iOS platform (iPhone, iPad, or iPod Touch).
```
Comment (hansemannn) [inline apidoc/Titanium/Platform/Platform.yml]: I would rephrase it to:

> Since 5.5.0.GA and Xcode 8, this property returns `iOS` for for the iOS platform (iPhone, iPad, or iPod Touch).

Are you fine with that? And good catch btw!

Comment (HazemKhaled) [inline apidoc/Titanium/Platform/Platform.yml]: done 👍

Comment (HazemKhaled) [inline apidoc/Titanium/Platform/Platform.yml]: I just noticed Ti.Platform.name return `iPhone OS` with Xcode 8 and iOS simulator 9.3, so it has to be.

```
On iOS 10+ return 'iOS' for for the iOS platform (iPhone, iPad, or iPod Touch).
```
